starboard/android/shared/log_flush.cc

```diff
@@ -17,5 +17,4 @@
 #include <stdio.h>
 #include <unistd.h>
 
-void SbLogFlush() {
-}
+void SbLogFlush() {}
```

Comment:
The most satisfying change in this PR, IMO.

Reply:
We do what we can :)